tests/unit/plugins/inventory/test_lxd.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Frank Dornheim <dornheim@posteo.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import pytest
+import os
+
+from ansible.errors import AnsibleError
+from ansible.inventory.data import InventoryData
+from ansible_collections.community.general.plugins.inventory.lxd import InventoryModule
+
+
+HOST_COMPARATIVE_DATA = {
+    'ansible_connection': 'ssh', 'ansible_host': '10.98.143.199', 'ansible_lxd_os': 'ubuntu', 'ansible_lxd_release': 'focal',
+    'ansible_lxd_profile': ['default'], 'ansible_lxd_state': 'running', 'ansible_lxd_location': 'Berlin',
+    'ansible_lxd_vlan_ids': {'my-macvlan': 666}, 'inventory_hostname': 'vlantest', 'inventory_hostname_short': 'vlantest'}
+GROUP_COMPARATIVE_DATA = {
+    'all': [], 'ungrouped': [], 'testpattern': ['vlantest'], 'vlan666': ['vlantest'], 'locationBerlin': ['vlantest'],
+    'osUbuntu': ['vlantest'], 'releaseFocal': ['vlantest'], 'releaseBionic': [], 'profileDefault': ['vlantest'],
+    'profileX11': [], 'netRangeIPv4': ['vlantest'], 'netRangeIPv6': ['vlantest']}
+GROUP_Config = {
+    'testpattern': {'type': 'pattern', 'attribute': 'test'},
+    'vlan666': {'type': 'vlanid', 'attribute': 666},
+    'locationBerlin': {'type': 'location', 'attribute': 'Berlin'},
+    'osUbuntu': {'type': 'os', 'attribute': 'ubuntu'},
+    'releaseFocal': {'type': 'release', 'attribute': 'focal'},
+    'releaseBionic': {'type': 'release', 'attribute': 'bionic'},
+    'profileDefault': {'type': 'profile', 'attribute': 'default'},
+    'profileX11': {'type': 'profile', 'attribute': 'x11'},
+    'netRangeIPv4': {'type': 'network_range', 'attribute': '10.98.143.0/24'},
+    'netRangeIPv6': {'type': 'network_range', 'attribute': 'fd42:bd00:7b11:2167:216:3eff::/24'}}
+
+
+@pytest.fixture(scope="module")
+def inventory():
+    inv = InventoryModule()
+    inv.inventory = InventoryData()
+
+    # Test Values
+    inv.data = inv.load_json_data('tests/integration/targets/inventory_lxd/files/lxd_inventory.atd')  # Load Test Data
```

Comment:
Move data files into a subdirectory `fixtures` of this directory, not to some other random place.

Reply:
done

Reply:
I do not see a directory `fixtures` in `tests/unit/plugins/inventory`.

Reply:
I misunderstood that. -> done